### ISSUE TITLE
Android: Editing issues in the Android App 

### ISSUE THREAD

**Issue by andifor** [Fmstrat/ownnote#151]
Scrolling in the text field is nearly impossible:

It's bad on my mobile with a 4.5" screen. 
It's impossible ob my tablet with a 7.7" screen. 

This makes it nearly impossible to edit notes with more than 4 lines.

**Comment by Fmstrat** [comment]
@andifor I'm not sure I follow. Can you provide some specific details on what you are seeing? A screenshot may be useful as well. Thanks.

**Comment by andifor** [comment]
OK, here you see screenshots of my phone and tablet.

View the note is fine:
![screenshot_2015-06-04-20-08-30](https://cloud.githubusercontent.com/assets/4958403/7991421/46ed3854-0af8-11e5-9fea-83c1e69bb502.png)


Editing the note only shows only 4 lines. Not all screen estate is used:
![screenshot_2015-06-04-20-08-49](https://cloud.githubusercontent.com/assets/4958403/7991444/60f59ade-0af8-11e5-9657-a3fbb1724447.png)


Even if the keyboard is collapsed, the text area is not restored:
![screenshot_2015-06-04-20-09-31](https://cloud.githubusercontent.com/assets/4958403/7991433/586b2460-0af8-11e5-9568-a8d2ab592873.png)


And a screen shot of my tablet:
![screenshot_2015-06-04-20-21-25](https://cloud.githubusercontent.com/assets/4958403/7991450/6a95ab6a-0af8-11e5-9786-e8816a0c54da.png)


Note: using the cursor keys does not scroll the text. It can be scrolled manually, but the small text area does not help.

**Comment by andifor** [comment]
I think labeling this as a question is not correct. From the user point of view, it's a bug, but I leave it to you how to deal with such issues.

**Comment by Fmstrat** [comment]
Yes, that's definately an issue and I don't see that on any of my test devices. Can you provide the following information:
- Device versions
- Android versions
- ownNote for Android versions

Thanks.

**Comment by andifor** [comment]
Phone:
Galaxy Nexus, Android 4.3, OneNote 0.0.13

Tablet:
Samsung Galaxy Tab 7.7, Android 4.0.4, OneNote 0.0.13

Please let me know how I can further help to debug.

**Comment by Fmstrat** [comment]
OK, so when you changed your Xposed apps did it impact this at all? I'm not able to recreate this at all. I also notice that on the tablet the DPI looks really high, with a very small font. That combined with the fact that you're experiencing this on both your devices makes me think there is something going on unique to your configuration. I'd love to fix this, I just can't get a grip around what it is yet since I can't reproduce it in any emulator or on any device (I've tried about 9 different setups).

**Comment by qkzoo** [comment]
@andifor, are you running a custom rom?

**Comment by andifor** [comment]
OK, I've got it.

First, I don't run a custom rom, I use plain Android. But I use XPrivacy, an Xposed module to block access to individual Android functions for individual applications. For some reason, I blocked get-, init- and setUserAgentString. I guess without this access, the web browser/editor component could not initialise correctly.

It turns out not to be a bug in oneNote. I'm happy about that and sorry for all the trouble I stirred.
